File: core/src/main/kotlin/com/bing/lan/core/base/mapper/EmailActiveMapper.java
package com.bing.lan.core.base.mapper;

import com.bing.lan.core.base.domain.EmailActive;

import java.util.List;

public interface EmailActiveMapper {
	int deleteByPrimaryKey(Long id);

	int insert(EmailActive record);

	EmailActive selectByPrimaryKey(Long id);

	List<EmailActive> selectAll();

	int updateByPrimaryKey(EmailActive record);
	
	EmailActive selectByCode(String code);
}

File: core/src/main/kotlin/com/bing/lan/core/base/mapper/EmailActiveMapper.kt
package com.bing.lan.core.base.mapper

import com.bing.lan.core.base.domain.EmailActive

interface EmailActiveMapper {
    fun deleteByPrimaryKey(id: Long?): Int

    fun insert(record: EmailActive): Int

    fun selectByPrimaryKey(id: Long?): EmailActive

    fun selectAll(): List<EmailActive>

    fun updateByPrimaryKey(record: EmailActive): Int

    fun selectByCode(code: String): EmailActive
}